qiita_pet/support_files/doc/source/faq.rst

```diff
@@ -102,6 +102,41 @@ To update your unzip for most operating systems you can simply use your regular
 admin program. However, for Mac we suggest using
 `this version of unzip <ftp://ftp.microbio.me/pub/qiita/unzip>`__.
 
+Do you have general analytical questions?
+-----------------------------------------
+
+Normally these are: How can I test X factor in my samples? Why do I see this pattern?
+Which statistical method better fits my question?
+
+As you can imagine, you are not alone as this is a common problem while doing analysis.
+Thus, we suggest posting your question to the `QIIME2 Forum <https://forum.qiime2.org/>`__.
+This will generally ensure that your question is answered in a timely manner. There
+are many users and developers monitoring the QIIME2 Forum. Additionally, posting to
+the forum also helps us to manage our support burden as other users may have the
```

Comment:
Suggestion:
Additionally, posting to the forum also helps us to manage our support burden as other users may have the same question in the future. -> Posting questions in the forum allows you to share answers with others, who may have similar questions in the future.